Makefile

```diff
@@ -1,18 +1,20 @@
+KUBE_API_VERSION?=1.25
+
 .PHONY: build
 build:
-	cargo build --release
+	K8S_OPENAPI_ENABLED_VERSION=$(KUBE_API_VERSION) cargo build --release
```

Comment:
Are you sure this is needed? We had to do that with libraries because we cannot specify the k8s version via the feature flag of k8s-openapi. But with kwctl/policy-server we can do that, and it always worked fine